src/libraries/Native/Unix/System.Native/pal_process.c

```diff
@@ -653,6 +653,47 @@ int32_t SystemNative_SetRLimit(RLimitResources resourceType, const RLimit* limit
 
 int32_t SystemNative_Kill(int32_t pid, int32_t signal)
 {
+#if defined(TARGET_MIPS64)
```

Comment:
I would be better to define `PAL_...` specific enum in pal_process.h (similar to other enums that exist there already). This enum should have just the few signals that we actually need, and this remapping can then be done unconditionally, without hardcoding of magic numbers.

Reply:
I'm worried that more signals will be added in Interop.Kill.cs (internal enum Signals:int{..}) later.

Reply:
If that happens, both places will need to be updated. It is same with all other PAL enums.

Reply:
ok